src/gen_kotlin/templates/module.kt

```diff
@@ -62,12 +62,13 @@ class BreezSDKModule(reactContext: ReactApplicationContext) : ReactContextBaseJa
     {% endif -%}
     {%- endfor %}  
     @ReactMethod
-    fun setLogStream(promise: Promise) {
+    fun setLogStream(filterLevel: String, promise: Promise) {
         executor.execute {
             try {
                 val emitter = reactApplicationContext.getJSModule(RCTDeviceEventEmitter::class.java)
+                val levelFilter = asLevelFilter(filterLevel);
```

Comment:
and then converted if set
```suggestion
                val levelFilter = filterLevel?.let { asLevelFilter(filterLevel) }
```